.travis.yml

```diff
@@ -17,14 +17,17 @@ env:
   - THREADSAFE=true DB=postgresql
   - THREADSAFE=false DB=postgresql
 gemfile:
+  - gemfiles/Gemfile.rails-5.0-stable
   - gemfiles/Gemfile.rails-4.2-stable
   - gemfiles/Gemfile.rails-4.1-stable
   - gemfiles/Gemfile.rails-4.0-stable
   - gemfiles/Gemfile.rails-3.2-stable
   - Gemfile
 matrix:
   allow_failures:
-    - gemfile: Gemfile # Not supporting rails 5 at the moment
+    - rvm: 2.0.0
+    - rvm: 2.1.0
```

Comment:
I'm not sure it's possible based on the docs (the examples only mirror what's in the PR currently).